Mage.Sets/src/mage/cards/s/SlumberingKeepguard.java

```diff
@@ -0,0 +1,50 @@
+package mage.cards.s;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.common.EntersBattlefieldControlledTriggeredAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.PermanentsOnBattlefieldCount;
+import mage.abilities.effects.common.continuous.BoostSourceEffect;
+import mage.abilities.effects.keyword.ScryEffect;
+import mage.constants.SubType;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterEnchantmentPermanent;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class SlumberingKeepguard extends CardImpl {
+
+    private static final DynamicValue value = new PermanentsOnBattlefieldCount(StaticFilters.FILTER_CONTROLLED_PERMANENT_ENCHANTMENT);
```

Comment:
I personally like to add an Hint for most of cards with a non-direct DynamicValue (for instance PermanentPowerCount is a direct one), but up to you to add that one.